packages/recommend/src/__tests__/getTrendingItems.test.ts

```diff
@@ -0,0 +1,44 @@
+import { TestSuite } from '../../../client-common/src/__tests__/TestSuite';
+
+const recommend = new TestSuite('recommend').recommend;
+
+function createMockedClient() {
+  const client = recommend('appId', 'apiKey');
+  jest.spyOn(client.transporter, 'read').mockImplementation(() => Promise.resolve());
+
+  return client;
+}
+
+describe('getTrendingItems', () => {
+  test('builds the request', async () => {
+    const client = createMockedClient();
+
+    await client.getTrendingItems(
+      [
+        {
+          indexName: 'products',
+        },
+      ],
+      {}
```

Comment:
Is it also added to other tests? it shouldn't be required to pass the empty object

Reply:
You can ignore my comment, thinking again, it allow us to ensure the body is empty so it's fine